scripts/release.py

```diff
@@ -359,6 +361,25 @@ def prs_by_email(start_ref, end_ref):
     The dictionary is alphabetically ordered by email address
     The pull request list is ordered by merge date
     """
+    # `emails` maps from other_emails to primary email, based on people.yaml.
+    emails = {}
+    try:
+        people_resp = requests.get(PEOPLE_YAML)
+        if people_resp.status_code != 200:
```

Comment:
Seems like you'd be better off calling `people_resp.raise_for_status()`, which will raise an appropriate Python exception if the HTTP status code indicates a failure. If the HTTP status code indicates a success, it will do nothing. [(docs)](http://docs.python-requests.org/en/latest/api/?highlight=raise_for_status#requests.Response.raise_for_status)